source/features/tags-on-commits-list.tsx

```diff
@@ -86,7 +86,7 @@ async function getTags(lastCommit: string, after?: string): Promise<CommitTags>
 async function init(): Promise<void | false> {
 	const cacheKey = `tags:${getRepo()!.nameWithOwner}`;
 
-	const commitsOnPage = $$('.list-view-item');
```

Comment:
Make sure to include test URLs for each feature changed.

e.g. https://github.com/refined-github/refined-github/commits/19.5.21.1921